examples/bindeps/foo/Cargo.lock

```diff
@@ -0,0 +1,7 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This `Cargo.lock` file is unnecessary, since `./foo` is a path dependency it will automatically get promoted to a workspace member (i.e. it's packages area already present in the parent `Cargo.lock` file)

Reply:
Hmm, good catch, it should not have been here.